.editorconfig

```diff
@@ -0,0 +1,24 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+indent_style = tab
+
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.{json,yml}]
+indent_style = space
+indent_size = 2
```

Comment:
2 spaces indentation in YAML files because that's conventional. As for JSON files, it tends to be easier to reason about prettified JSON files with 2 space indentation. It's also what `npm` uses per default for its `package.json`.